func.rb

```diff
@@ -1,7 +1,7 @@
 require 'fileutils'
 
 class Func < Formula
-  v = "v0.25.1"
+  v = "knative-v1.7.0"
```

Comment:
I think you just need the `v1.7.0` here

Reply:
Yup

Reply:
But, when I looked at the link for downloading the binary is using that 
- https://github.com/knative-sandbox/kn-plugin-func/releases/tag/knative-v1.7.0
- https://github.com/knative-sandbox/kn-plugin-func/releases/download/knative-v1.7.0/func_darwin_amd64

@lance @maximilien

Reply:
Yes I see that now. It's odd to me that most of the plugins use basic semver, but the `kn` client is the same, using `knative-vX.X.X`. I don't really understand why the difference. 🤷